Assets/MixedRealityToolkit/Interfaces/InputSystem/IMixedRealityRaycastProvider.cs

```diff
@@ -19,12 +19,18 @@ public interface IMixedRealityRaycastProvider : IMixedRealityDataProvider
         /// <summary>
         /// Performs a raycast using the specified <see cref="Microsoft.MixedReality.Toolkit.Physics.RayStep"/>.
         /// </summary>
+        /// <remarks>
+        /// If nothing was hit, hitInfo will be null.
```

Comment:
We're already using the bool / out pattern that Unity uses for raycast hits. Shouldn't we just return false if we hit nothing, instead of making hitInfo null?